apps/modernization-ui/src/apps/search/investigation/CriteriaSearchFields.tsx

```diff
@@ -0,0 +1,115 @@
+import { SelectInput } from 'components/FormInputs/SelectInput';
+import { handleChangeToDefaultValue } from 'forms/event';
+import { CaseStatus, InvestigationStatus, NotificationStatus, ProcessingStatus } from 'generated/graphql/schema';
+import { UseFormReturn, Controller } from 'react-hook-form';
+import { formatInterfaceString } from 'utils/util';
+import { InvestigationFilterEntry } from './InvestigationFormTypes';
+import { MultiSelectInput } from 'components/selection/multi';
+
+type Props = {
+    form: UseFormReturn<InvestigationFilterEntry>;
+};
+
+const CriteriaSearchFields = ({ form }: Props) => {
+    return (
+        <>
+            <Controller
+                control={form.control}
+                name="investigationStatus"
+                render={({ field: { onChange, value, name } }) => (
+                    <SelectInput
```

Comment:
suggestion:  The new `SingleSelect` component works with `Selectable` so you don't have to default the blank `Selectable` objects and the `onChange` from the hook can be passed directly to it.